hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/OzoneClientConfig.java

```diff
@@ -124,6 +125,14 @@
       tags = ConfigTag.CLIENT)
   private boolean checksumVerify = true;
 
+  @Config(key = "checksum.combine.mode",
+      defaultValue = "MD5MD5CRC",
```

Comment:
Could you change the default value to COMPOSITE_CRC. This will make things easy during the file checksum comparison and IMHO its not required to go with HDFS default behavior.